misc/open_pdks/meta.yaml

```
@@ -0,0 +1,56 @@
+# Use `conda-build-prepare` before building for a better version string.
+{% set version = '%s_%04i_%s'|format(GIT_DESCRIBE_TAG|replace('open_pdks-', '')|replace('-', '') or '0.X', GIT_DESCRIBE_NUMBER|int, GIT_DESCRIBE_HASH or 'gUNKNOWN') %}
```

```suggestion
{% set version = '%s_%04i_%s'|format(GIT_DESCRIBE_TAG|replace('-', '') or '0.X', GIT_DESCRIBE_NUMBER|int, GIT_DESCRIBE_HASH or 'gUNKNOWN') %}
```
Having such a replace only makes sense when tags in repository are prefixed with a name but it isn't the case for `open_pdks`. I omitted this before but it can stay, it's not a serious blocker.

---

Done